package.json

```diff
@@ -16,9 +16,10 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
-    "link-check-diff": "./scripts/link-check-git-diff.sh",
-    "link-check-exclude": "./scripts/exclude-links-check.sh"
+    "link-check": "repo-link-check -c config/link-check/config.yml -s filesystem -u",
```

Comment:
reminder: probably need to check and fix the docs contrib guide if it mentions links check

Reply:
A case-insensitive ripgrep for `link.?check` shows that no docs mention the link check, so I think we're good here.